config/locales/devise/devise.de_formal.yml

```diff
@@ -57,6 +57,7 @@ de_formal:
     passwords:
       edit:
         change_password: "Mein Passwort ändern"
+        pam_account: "Eine Passwortänderung ist nicht möglich - Die Account Authentifizierung erfolgt durch PAM"
```

Comment:
Please do not edit any non-English locale, these changes will simply get lost with the next locale update.

Reply:
thx, didn't know